bundles/AdminBundle/Resources/public/js/pimcore/object/classes/data/calculatedValue.js

```diff
@@ -56,7 +56,7 @@ pimcore.object.classes.data.calculatedValue = Class.create(pimcore.object.classe
                 ]
             },
             {
-                xtype: "numberfield",
+                xtype: "textfield",
```

Comment:
this means user needs to provide a unit as well. as width will be treated as css value https://docs.sencha.com/extjs/6.0.0/classic/Ext.form.field.Text.html#cfg-width.. so there should be info for the user that how to provide width values

Reply:
I add a explanation to every width textfield